site/installing.md

```diff
@@ -240,14 +240,40 @@ Allowable parameters for the launcher URL:
 
 To download and read the Scope manifest run:
 
-    curl --silent -L --remote-name https://cloud.weave.works/launch/k8s/weavescope.yaml
+    curl --silent --remote-name https://cloud.weave.works/k8s/scope.yaml
```

Comment:
Don't you also need --location just in case there's a redirect?
And .. about --remote-name, why is it needed? Shouldn't it be fixed in the service?